mysql-test/include/have_rocksdb_as_default.inc

```diff
@@ -0,0 +1,12 @@
+--source include/have_rocksdb.inc
```

Comment:
Maybe push --default-storage-engine=rocksdb to suite.opt too? and then have_rocksdb_as_default.inc becomes redundant? But it needs to be checked what --d-s-e does for an unavailable SE, so that we don't silently test InnoDB, and I'm not sure what to do about `set global rocksdb_force_flush_memtable_now = true;`